userprofile/tasks.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015-2020 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Worker tasks for user-related features
+"""
+
+# Imports #####################################################################
+
+import logging
+
+from django.conf import settings
+from huey.api import crontab
+from huey.contrib.djhuey import db_periodic_task
+from mailchimp3 import MailChimp
+from more_itertools import chunked
+
+from userprofile.models import UserProfile
+
+
+# Logging #####################################################################
+
+logger = logging.getLogger(__name__)
+
+
+# Tasks #######################################################################
+
+@db_periodic_task(crontab(day='*/1', hour='2', minute='0'))
```

Comment:
It might be, as this was me doing cargo cult programming :/

I've picked up this "pattern" from the [instance.tasks](https://github.com/open-craft/opencraft/blob/petarmaric/bb-2236-add-trial-users-to-mailchimp-list/instance/tasks.py) module, while constructing the OpenCraft file header and mimicking our code style.

Fixed in c639ba6